packages/form-js-viewer/test/spec/util/GetSchemaVariables.spec.js

```diff
@@ -0,0 +1,27 @@
+import {
+  getSchemaVariables
+} from '../../../src';
+
+import schema from '../form.json';
+import dynamicSchema from '../dynamic.json';
+
+describe('extractSchemaVariables', () => {
```

Comment:
```suggestion
describe('util/getSchemaVariables', () => {
```

Or as an alternative: just have a `Util.spec.js` and have

```js
describe('util', function() {
  
  describe('#getSchemaVariables');
  
  // others we want to test
});
```

Reply:
This might get a little more complex when we start doing more variable dependent stuff, like conditional rendering and such. So I think it deserves its own file.